build/lib/stylelint/vscode-known-variables.json

```diff
@@ -647,6 +647,9 @@
 		"--vscode-tab-activeBackground",
 		"--vscode-tab-activeBorder",
 		"--vscode-tab-activeBorderTop",
+		"--vscode-tab-selectedBorderTop",
```

Comment:
Both exist and because `selected` sounds a bit better in this context I prefer keeping it like this
`"--vscode-editorSuggestWidget-selectedBackground",
		"--vscode-editorSuggestWidget-selectedForeground",
		"--vscode-editorSuggestWidget-selectedIconForeground",`